bigquery/README.md

```diff
@@ -4,6 +4,18 @@ This notifier pushes build data to a BigQuery instance.
 
 *Alpha Feature - Under Development*
 
+## Dataset and table setup
+
+The BQ notifier initializes datasets and tables automatically.
+Dataset identifiers in the BigQuery Notifier config can refer to existing or nonexistent datasets. 
+
+Table identifiers in the BigQuery Notifier config can refer to either:
+1. A nonexistent table (will be created upon deployment of the notifier)
+2. An empty table not yet initialized with a schema.
```

Comment:
FYI - it's a weird Markdown feature, but if you just put `1.` in front of all of these, GitHub will format the ordering for you.

Reply:
Up to you if you want to do this.